pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/data/repository/UserRepository.java

```diff
@@ -94,6 +90,7 @@ public User findOrCreate(User inputUser) {
             user = createUser(inputUser);
         } catch (NonUniqueResultException e) {
             logger.error("findOrCreate() " + e.getClass().getSimpleName() + ": " + e.getMessage());
+            user = createUser(inputUser);
```

Comment:
This doesn't make sense to me. What are you fixing by creating an additional user when the subject already maps to at least two users?